### ISSUE TITLE
Using the containee image with a different size of the container

### ISSUE THREAD

**Issue by mghmgh1281375** [mhsattarian/steganography#1]
Please add support for encoding an image with a different **size** of the container.

**Comment by mhsattarian** [comment]
That's right. it needs container and containee images to be the same size. should've mentioned that in Readme.
pull requests would be appreciated.
